modules/op-console-statistics.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assemblies:
+// * create/working-with-pipelines-web-console.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-console-statistics_{context}"]
+= Pipeline execution statistics in the web console
+
+You can view statistics related to execution of pipelines in the web console.
+
+To view the statistic information, you must first enable the {pipelines-shortname} console plugin.
+
+Statistic information is available for all pipelines together and for each individual pipeline.
```

Comment:
Fine to leave as is, but it looks a little odd with three sentences each on their own. Just wondering if it might look better if one of them was doubled up in the same paragraph as another. Again, fine to leave as is though if you prefer.